### PR TITLE
User window settings

### PR DESCRIPTION
**Pull request by vlaskiri**

### CODE REVIEW

File: src/components/chatWindow/WindowChat.jsx [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
-    const serverUser = useSelector(selectUser)
-    
+    const dispatch = useDispatch();
+    const user = JSON.parse(localStorage.getItem('user'));
+    const serverUser = useSelector(selectUser);
+    const isOpenChat = useSelector(selectOpenChat);
+    const {i18n, t} = useTranslation();
+
+    // User Language
+    const languageList = {
```
Comment (elzbthbrst) [inline src/components/chatWindow/WindowChat.jsx:19]: r u shure u need this vocabulary here?

File: src/components/chatWindow/navigation/nav-modal/settings-modal/SettingsLanguages.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {MdOutlineLanguage} from 'react-icons/md';
+import {useTranslation} from 'react-i18next';
+
+const locales = {
```
Comment (elzbthbrst) [inline src/components/chatWindow/navigation/nav-modal/settings-modal/SettingsLanguages.jsx:5]: the same question

File: src/styles/chat-window/navigation/settings-modal.scss [hunk 3]
```diff
@@ -2,100 +2,232 @@
...
+        // .settings__title
+        color: $color-black;
         display: flex;
-        align-items: center;
         justify-content: space-between;
-        margin-bottom: 30px;
+        align-items: center;
+        margin-bottom: 24px;
+
+            h2 {
```
Comment (elzbthbrst) [inline src/styles/chat-window/navigation/settings-modal.scss:59]: why u dont use className text-inter-18-600 from static-classes?

File: src/styles/chat-window/navigation/settings-modal.scss [hunk 4]
```diff
@@ -2,100 +2,232 @@
...
+        margin-left: 16px;
+        width: 100%;
+
+            input {
+                width: 100%;
+                padding: 6px 0;
+                border: none;
+                border-bottom: 1px solid $color-black;
+                color: $color-black;
+                font-family: $font-inter;
```
Comment (elzbthbrst) [inline src/styles/chat-window/navigation/settings-modal.scss:100]: static-classes

File: src/components/chatWindow/WindowChat.jsx (hunk 1 above)
Comment (vlaskiri) [reply src/components/chatWindow/WindowChat.jsx:19]: I moved it to the extra/config folder

**Review by elzbthbrst — APPROVED** [review]

**Pull request merged**